tracer/README.MD

```diff
@@ -1,20 +1,115 @@
 ﻿# dd-trace-dotnet/tracer
 
-This repo ([dd-trace-dotnet](https://github.com/DataDog/dd-trace-dotnet)) contains the sources for the client-side components of the Datadog product suite for Application Telemetry Collection and Application Performance Monitoring for .NET Applications.
+This folder contains the sources of the Datadog .NET APM tracer. It automatically instruments supported libraries out-of-the-box and also supports manual instrumentation to instrument your own code.
```

Comment:
In public docs, we use capitalized "Tracer" and "Agent". There was also a big push some time ago to replace "manual" instrumentation with "custom" instrumentation.

```suggestion
This folder contains the sources of the Datadog .NET APM Tracer. It automatically instruments supported libraries out-of-the-box and also supports custom instrumentation to instrument your own code.
```